cryptography/src/bls12381/primitives/group.rs

```diff
@@ -203,6 +203,11 @@ impl Scalar {
         Self(ret)
     }
 
+    /// Creates a new scalar from the provided index (a scalar offset by 1).
+    pub fn from_index(i: u32) -> Self {
```

Comment:
Decided to create a proper initializer for `indices` (that takes care of offset internally).